skipruntime-ts/core/src/api.ts

```diff
@@ -407,7 +407,7 @@ export type CollectionUpdate<K extends Json, V extends Json> = {
  * If it _is_ a Skip service, then `SkipExternalService` can be used, specifying an entrypoint.
```

Comment:
(I wanted to comment on 2 lines above but github says fu)

I think it would be better to have a sentence that emphasizes that user-defined ExternalServices are expected. Maybe something like the following:

> Custom implementations of the `ExternalService` interface can be defined as needed, and there are two implementations provided by the @skipruntime/helpers package: `SkipExternalService` and `PolledExternalService`.